scripting/include/srccoop_api/classdef/core/CSoundEnt.inc

```diff
@@ -1,11 +1,19 @@
 #pragma newdecls required
 #pragma semicolon 1
 
+static int g_iCSoundSize = -1;
+
 methodmap CSoundEnt < CPointEntity
 {
+	public static void InitClassdef(const GameData hGameConfig)
+	{
+		g_iCSoundSize = GetGamedataInt(hGameConfig, "CSoundSize");	// TODO: Change this to `hGameConfig.GetOffset`.
```

Comment:
In other libraries I used sizes are defined as offsets like so `"sizeof(CSound)"`

Reply:
I like that a lot.

Changed the variable assignment to use `hGameConfig.GetOffset` and renamed gamedata variable to `sizeof(CSound)`.